EIPS/eip-894.md

```diff
@@ -0,0 +1,39 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Definition
+The definition of bailouts in this EIP is state changes to the Ethereum blockchain, through a hardfork, that treat account holders in different ways.
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
+
+
+## Abstract
+Proposals that advocate for bailouts on the Ethereum blockchain should not be merged into the official Ethereum repositories.
+
+
+## Motivation
+The issue of bailouts on the Ethereum blockchain is always controversial. This EIP attempts to raise the barriers for bailouts.
```

Comment:
This is a bad assumption.  The controversial-ness of an irregular state change can be measured with many different (imperfect) mechanisms like CarbonVote, miner signalling, etc.  It would be a bad decisions to assert that *anything* is "always" controversial.  

This EIP doesn't raise the barrier for bailouts, it raises the barrier for discussing what you consider to be "bailouts".  If approved as intended, it would be nothing more than frivolous censorship of harmless discussion. 

Opposition to irregular state changes should be voiced in a fair setting, which this proposal seeks to undermine.

Reply:
It is not censorship since it is not deleted or restricted in any way. It is just not merged.

Reply:
How is refusal to merge on principle *not* arbitrary restriction?

Reply:
There are no restriction on freedom of speech.

Reply:
I really don't think this EIP or the one it's clearly in response to have anything to do with freedom of speech.

Reply:
Finally something we agree on :)